app/importers/file_importers/student_section_grades_importer.rb

```diff
@@ -0,0 +1,31 @@
+class StudentSectionGradesImporter < Struct.new :school_scope, :client, :log, :progress_bar
+  def initialize(*)
+    super
+    self.student_lasid_map = Student.all.pluck(:local_id,:id).to_h
+    self.section_number_map = ActiveRecord::Base.connection.execute("select CONCAT(sections.section_number, '|', sections.term_local_id, '|', schools.local_id, '|', courses.course_number), sections.id  from sections inner join courses on courses.id = sections.course_id inner join schools on schools.id = courses.school_id").values.to_h
```

Comment:
Can you say more about writing SQL here rather than Ruby?  It seems like adding `.includes(course: school)` and then mapping over them to concat like you mentioned would be just fine.  All things being equal, I think it's good to default to the idioms of the language, framework and the codebase, but of course we should also do what makes sense if there are good reasons to go our own way!

Reply:
Yeah, it will be fine to map over them to concat.